pajbot/apiwrappers/bttv.py

```diff
@@ -49,8 +49,7 @@ def fetch_channel_emotes(self, channel_id):
             if e.response.status_code == 404:
                 # user does not have any BTTV emotes
                 return []
-            else:
-                raise e
+            raise e
```

Comment:
Let's add a little space
```suggestion

            raise e
```